src/Core/src/Platform/Windows/KeyboardAcceleratorExtensions.cs

```diff
@@ -11,18 +11,13 @@ public static class KeyboardAcceleratorExtensions
 	{
 		public static void UpdateAccelerator(this MenuFlyoutItemBase platformView, IMenuFlyoutItem menuFlyoutItem)
 		{
-#if NETCOREAPP || NETSTANDARD2_1_OR_GREATER
-			var keyboardAccelerators = menuFlyoutItem.Accelerator?.ToPlatform();
-#else
-			var keyboardAccelerators = menuFlyoutItem.Accelerator.ToPlatform();
-#endif
+			var keyboardAccelerators = menuFlyoutItem.Accelerators?[0].ToPlatform();
```

Comment:
What happens if there is 0 items - but not null? Will it throw? Maybe we need a:

```suggestion
			var keyboardAccelerators = menuFlyoutItem.Accelerators?.FirstOrDefault().ToPlatform();
```

Reply:
If Windows support multiple acc, then maybe we can just implement here? Just use this new feature we got.

Reply:
I had the same thought at first, but I can't do that without a `!` because of the ToPlatform. Which is less worse??

Reply:
@jsuarezruiz thoughts on supporting multiple accelerators just for windows?

Reply:
Added support to multiple accelerators just for Windows.